testing/zerotier-one/zerotier-one.pre-install

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+/usr/sbin/addgroup -S zerotier-one
+/usr/sbin/adduser -D -H -S -h /var/lib/zerotier-one -s /sbin/nologin -g 'https://www.zerotier.com/' -G zerotier-one zerotier-one
```

Comment:
The GECOS is probably not needed, and `-s /sbin/nologin` and `-D` are implicit for system (`-S`) accounts, they can be omitted. Finally, if you omit `-G zerotier-one` you can omit the `addgroup` call above entirely. :)